### ISSUE TITLE
关于运行时间问题

### ISSUE THREAD

**Issue by gaoshihao333** [THUDM/HGB#58]
再次叨扰！请问论文中的运行时间指的是，每个epoch中的训练时间吗？例如本篇文章所提出的模型其中的代码：
            t_start = time.time()
            # training
            net.train()
            logits = net(features_list, e_feat)
            logp = F.log_softmax(logits, 1)
            train_loss = F.nll_loss(logp[train_idx], labels[train_idx])
            optimizer.zero_grad()
            train_loss.backward()
            optimizer.step()
            t_end = time.time()
如果是的话，我跑出来时间大约是0.05秒，与论文中的图上时间不符。另外相同的情况，只看每个epoch的训练时间HAN模型也只要0.04秒左右，与论文中大约7、8秒一个epoch也不符合。如有时间望回复！非常感谢！！！

**Comment by 1049451037** [comment]
不同的硬件跑的速度可能不一样，但是0.04秒一个epoch也太快了吧……一个epoch是指遍历完所有数据一遍，而不是一个step。

**Comment by gaoshihao333** [comment]
> 不同的硬件跑的速度可能不一样，但是0.04秒一个epoch也太快了吧……一个epoch是指遍历完所有数据一遍，而不是一个step。

按照上述代码打印出的时间不是每个epoch的时间吗？...

**Comment by 1049451037** [comment]
哦，确实是的。那可能是硬件不同、dgl版本更新等因素导致的，我当时做实验是3年前了，显卡可能比较旧。

**Comment by gaoshihao333** [comment]
您提出模型代码片段 和 时间结果：
![image](https://github.com/THUDM/HGB/assets/116276246/fe452ad0-0722-4925-b3e7-c773b2b380d4)
<img width="273" alt="image" src="https://github.com/THUDM/HGB/assets/116276246/8fb24718-9b37-44eb-b57d-fba489c4d02b">

HAN模型代码片段 和 时间结果：
![image](https://github.com/THUDM/HGB/assets/116276246/1b8db71d-6bfa-4be0-a3e6-fccd9b5c299e)
![image](https://github.com/THUDM/HGB/assets/116276246/06c7bec4-a1e9-4704-9171-122e36ddab91)
以上代码都是您文件的代码，HAN的打印时间是我加上的，您看有问题吗？  
耽误您时间了 感谢！

**Comment by 1049451037** [comment]
没问题

**Comment by gaoshihao333** [comment]
> 没问题

好的，感谢大佬！Have a good day！祝新年快乐！
